<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background: url(:/i) no-repeat right bottom;
 	background-size: cover;
	background-attachment: fixed;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="line_logs">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>291</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="button_insert">
         <property name="text">
          <string>Insert</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_insert">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="button_remove">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_remove">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="button_search">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_search">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="button_add_subtree">
         <property name="text">
          <string>Add subtree</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_subtree">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>372</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>button_insert</sender>
   <signal>clicked()</signal>
   <receiver>line_insert</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_remove</sender>
   <signal>clicked()</signal>
   <receiver>line_remove</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_search</sender>
   <signal>clicked()</signal>
   <receiver>line_search</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_add_subtree</sender>
   <signal>clicked()</signal>
   <receiver>line_subtree</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
